Real_Estate/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Real Estate",
+    "summary": "Real Estate Module helps to buy and sell Properties.",
+    "description": "",
+    "application": True,
+    "category": "Tutorials",
+    "installable": True,
+    "depends": ['base'],
+    "data": ['security/ir.model.access.csv', 'views/real_estate_property_views.xml', 'data/real_estate_property_data.xml', 'views/real_estate_menu_views.xml'],
```

Comment:
```suggestion
    "data": [
        'security/ir.model.access.csv',
        'views/real_estate_property_views.xml',
        'data/real_estate_property_data.xml',
        'views/real_estate_menu_views.xml'
    ]
```
Please follow [PEP8](https://peps.python.org/pep-0008/#maximum-line-length) Guidelines